modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/price/test/CommerceProductPriceCalculationV2Test.java

```diff
@@ -1750,10 +1757,17 @@ public void testCalculatePriceStaticOptionSKUBundle() throws Exception {
 					StringPool.BLANK));
 
 		unitPrice = commerceProductPrice.getUnitPrice();
+
+		unitPricePrice = unitPrice.getPrice();
+
 		finalPrice = commerceProductPrice.getFinalPrice();
 
-		Assert.assertEquals(BigDecimal.valueOf(13), unitPrice.getPrice());
-		Assert.assertEquals(BigDecimal.valueOf(26), finalPrice.getPrice());
+		finalPricePrice = finalPrice.getPrice();
+
+		Assert.assertEquals(
+			BigDecimal.valueOf(13), unitPricePrice.stripTrailingZeros());
```

Comment:
I think this should go between unitPricePrice and finalFinal (like you did on line 1744) - otherwise are different